plugins/source/hubspot/client/spec.go

```diff
@@ -1,10 +1,12 @@
 package client
 
 type Spec struct {
-	MaxRequestsPerSecond *int                     `yaml:"max_requests_per_second,omitempty" json:"max_requests_per_second,omitempty"`
-	TableOptions         map[string]*TableOptions `yaml:"table_options,omitempty" json:"table_options,omitempty"`
+	MaxRequestsPerSecond *int           `yaml:"max_requests_per_second,omitempty" json:"max_requests_per_second,omitempty"`
+	TableOptions         TablesSettings `yaml:"table_options,omitempty" json:"table_options,omitempty"`
```

Comment:
```suggestion
	TablesSettings         TablesSettings `yaml:"table_options,omitempty" json:"table_settings,omitempty"`
```
for consistency (or the other way around)